lib/src/chat_view.dart

```diff
@@ -352,7 +352,8 @@ class DashChatState extends State<DashChat> {
     textController = widget.textController ?? TextEditingController();
 
     Timer(Duration(milliseconds: 500), () {
-      scrollController.jumpTo(scrollController.position.maxScrollExtent);
+      double initPos = widget.inverted ? 0.0 : scrollController.position.maxScrollExtent;
```

Comment:
Thanks for the pull request, can you make this change and let me know I will update the package later

```suggestion
      widget.inverted ? 0.0 : scrollController.position.maxScrollExtent;
```